### ISSUE TITLE
Rustic cannot read snapshots created with restic 0.17.0

### ISSUE THREAD

**Issue by 9-2-1** [rustic-rs/rustic_core#244]
```
> rustic -P config snapshots
[INFO] using config config.toml
[INFO] repository F:\restic-repo: password is correct.
[INFO] using cache at C:\Users\Legion\AppData\Local\rustic\e5e8b63342b9d4ad2db6cd90683754ee887d28d74a6b806f26763c6b2edaf015
error: deserializing from bytes of JSON Text failed: `Error("missing field `total_dirs_processed`", line: 1, column: 575)`
```

Likely because that restic 0.17.0 create summary information like rustic but without a `total_dirs_processed` field.

```
> restic version
restic 0.17.0 compiled with go1.22.5 on windows/amd64
> rustic -v
rustic v0.7.0
```

**Comment by aawsome** [comment]
Try the lastest nightly builds. This is already fixed and rustic 0.8.0 will be released soon...

**Comment by aawsome** [comment]
0.8.0 has been released.
